GitCommands/DiffMergeTools/WinMerge.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+
+namespace GitCommands.DiffMergeTools
+{
+    internal class WinMerge : DiffMergeTool
+    {
+        /// <inheritdoc />
+        public override string DiffCommand => "-e -u \"$LOCAL\" \"$REMOTE\"";
+
+        /// <inheritdoc />
+        public override string ExeFileName => "winmergeu.exe";
+
+        /// <inheritdoc />
+        public override string MergeCommand => "-e -u -dl \"Original\" -dr \"Modified\" \"$MERGED\" \"$REMOTE\"";
```

Comment:
case "winmerge":
                    return "\"" + exeFile + "\" -e -u  -wl -wr -fm -dl \"Mine: $LOCAL\" -dm \"Merged: $BASE\" -dr \"Theirs: $REMOTE\" \"$LOCAL\" \"$BASE\" \"$REMOTE\" -o \"$MERGED\"";

Reply:
For a different version we need to add another `DiffMergeTool` definition

Reply:
This was a change in the defaults

Reply:
Thank you